### PR TITLE
PLAYER-1377 - load iq in core by default

### PR DESCRIPTION
**Pull request by dallen42**
add switch to allow player / provider metadata to turn IQ plugin events on / off

### CODE REVIEW

**Comment by aeng7** [comment]
@dallen42 , can we add some unit tests for this? LGTM otherwise

**Comment by dallen42** [comment]
@aeng7 unit tests are added

**Comment by aeng7** [comment]
LGTM, need one more review to approve and merge. @pilievOoyala @jpfranco

File: test/unit-tests/testIq.js [hunk 1]
```diff
@@ -416,4 +425,51 @@ describe('Analytics Framework Template Unit Tests', function()
     //expect(plugin.getAutoPlay()).toBe(false);
   });
 
+  it('IQ Plugin should enable iq reporting when provider metadata is received', function()
+  {
+    var oldMetadata = {
```
Comment (pilievOoyala) [inline test/unit-tests/testIq.js]: Should we also test what the default is?

**Comment by pilievOoyala** [comment]
LGTM. Should we add a test for the default? Might be something we add later but since we are likely to change the default in the next month or two, we can probably refrain from it for now.  You can add the unit test in a separate checkin.

**Pull request merged**